### DISCUSSION TITLE
Arduino IDE

### DISCUSSION THREAD

**Discussion by danielducu** [PricelessToolkit/MailBoxGuard#43] [Q&A]
What's the recommended Arduino IDE for this ? I'm with the latest 2.3.1 and getting all sort of errors ..

**Comment by PricelessToolkit** [comment]
For "megaTinyCore" the official recommendation is. Only versions of the Arduino IDE downloaded from https://arduino.cc  should be used, NEVER from a Linux package manager. the golden standard is "V1.8.13".

**Comment by danielducu** [comment]
> For "megaTinyCore" the official recommendation is. Only versions of the Arduino IDE downloaded from https://arduino.cc  should be used, NEVER from a Linux package manager. the golden standard is "V1.8.13".

Thanks. Works i managed to make it work with the latest version. It works hehe

**Comment by sanjulagathsara** [comment]
Try using a latest version
